apps/bcc3d.ca/pages/request.html

```diff
@@ -27,6 +27,12 @@
   - id: 6
     label: Pharmacy
   - id: 7
+    label: Dentist
+  - id: 8
+    label: Physio, Chiropractor or Massage
+  - id: 9
+    label: Non-Medical
+  - id: 10
```

Comment:
Yes, you'll need to add the entries to the QA OpenProject and test that they work in QA, then do the same thing when you deploy live. Be careful with the IDs here. I think the "Other" ID might stay static (at 7), and the others will get added with the later IDs.

Reply:
Thanks @wsonnex !
What credentials should I use to access the OpenProject on the QA and Live environment?
Thanks for the heads up regarding the "Other" type. I'll test on QA and confirm if it's static at ID 7.